src/util.cpp

```diff
@@ -268,7 +268,7 @@ void LogPrintStr(const std::string &str)
 
             // Debug print useful for profiling
             if (fLogTimestamps && fStartedNewLine)
-                fprintf(fileout, "%s ", DateTimeStrFormat("%x %H:%M:%S",  GetAdjustedTime()).c_str());
+                fprintf(fileout, "%s\n", DateTimeStrFormat("%x %H:%M:%S",  GetAdjustedTime()).c_str());
```

Comment:
The lines are now formatted as 
```
    time 
    message
````
I much rather prefer 
````
    time message
````

Reply:
Ah, good catch. I'll change it.

Reply:
Log timestamps would be a good thing to have enabled by default.